units/sys-fs-fuse-connections.mount

```diff
@@ -11,6 +11,7 @@ Documentation=https://www.kernel.org/doc/Documentation/filesystems/fuse.txt
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
 DefaultDependencies=no
 ConditionPathExists=/sys/fs/fuse/connections
+ConditionCapability=CAP_SYS_ADMIN
```

Comment:
so far nobody complained  about dev-mqueue.... I'd rather wait until somebody complains before we litter everything with conditions...